searchcore/src/vespa/searchcore/proton/attribute/attributemanager.cpp

```diff
@@ -583,6 +584,16 @@ AttributeManager::asyncForEachAttribute(std::shared_ptr<IConstAttributeFunctor>
     }
 }
 
+void
+AttributeManager::asyncForEachAttribute(std::shared_ptr<IAttributeFunctor> func) const
+{
+    for (const auto &attr : _attributes) {
+        AttributeVector::SP attrsp = attr.second.getAttribute();
```

Comment:
Missing check and skip if `attr.second.isExtra()` is true.

Reply:
Why should it be different for the extra attributes ?

Reply:
Because the extra attribute (document meta store) uses the document db main executor.

Reply:
Yes, that was the only extra one.